gemsiv/delaunay/quadedge.C

```diff
@@ -1,4 +1,9 @@
 #include "quadedge.h"
+#include "../../dummy.h"
```

Comment:
The code in question seems to be using IRIS GL which is not available. I tried to have it compiled by adding all the missing functions. Maybe I should rename dummy.[hc] to irisgl, fakegl or something like that. Do you have a suggestion?